src/errors/errors.d.ts

```diff
@@ -17,6 +18,6 @@ export declare const exceptions : ExceptionAliasSet & {
   Type: ErrorConstructor,
   Range: ErrorConstructor
 };
-export declare const errnames : {[P in keyof ExceptionSet]: P};
+export declare const errnames : {[P in keyof DexieErrors]: P};
```

Comment:
I think the correct typing here should be just:
```ts
export declare const errnames: DexieErrors;
```
...so that we get {Modify: "ModifyError"} rather than {Modify: "Modify"}.